x-pack/solutions/chat/plugins/wci-index-source/public/integration/configuration.tsx

```diff
@@ -50,6 +58,35 @@ export const IndexSourceConfigurationForm: React.FC<IntegrationConfigurationForm
   ];
 
   const { generateSchema } = useGenerateSchema();
+  const { isLoading, autocompleteQuery } = useIndexNameAutocomplete();
+
+  const [selectedOptions, setSelected] = useState([] as EuiComboBoxOptionOption[]);
+
+  const onChange = (selectedOptions) => {
+    setSelected(selectedOptions);
+  };
+
+  useDebounce(
+    () => {
+      const result: string[] = [];
+      setOptions(result);
```

Comment:
tech nit: `setOptions([]);`

func nit: Not sure we want to purge the suggestion list while fetching. I think the right practice is to just keep the stale items and replace them when we get the new ones